sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -58,9 +59,10 @@ type hostConnections struct {
 }
 
 type connStatus struct {
-	firstSeen timestamp.MicroTS
-	lastSeen  timestamp.MicroTS
-	used      bool
+	firstSeen   timestamp.MicroTS
+	lastSeen    timestamp.MicroTS
+	used        bool
+	usedProcess bool
```

Comment:
Don't delete an endpoint unless it has been used by both the regular networkflows path and processes listening on ports paths.

Reply:
Add this as the code commentary to the structure field, together with the details why keeping an endpoint is needed for PLOP correctness.

Reply:
Done